src/hr/dto/create-hr.dto.ts

```diff
@@ -0,0 +1 @@
+export class CreateHrDto {}
```

Comment:
Po co tworzysz `CreateHrDto` skoro nic tu nie dodajesz?:)
Powinien tu być: email, fullName, company, MaxReservedStudents to chyba potem sobie HR ustawia, nie?
Możesz się wzorować na `CreateUserDto` jeśli jeszcze nie znasz class-validator.

Przemyśl tylko temat hasła. Admin go nie dodaje, ale potem HR sobie je przy rejestracji uaktualnia, więc to pozostaje raczej w kwestii modułu `Users`